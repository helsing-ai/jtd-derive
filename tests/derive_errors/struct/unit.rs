#[derive(jtd_derive::JsonTypedef)]
struct Unit;

fn main() {}
