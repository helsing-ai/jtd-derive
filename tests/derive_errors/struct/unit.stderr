error: jtd-derive does not support unit structs
 --> tests/derive_errors/struct/unit.rs:2:8
  |
2 | struct Unit;
  |        ^^^^
